istio-2-mixin/panels.libsonnet

```diff
@@ -0,0 +1,1061 @@
+          table.fieldOverride.byName.new('HTTP/GRPC tx delay')
+          + table.fieldOverride.byName.withProperty('custom.width', 157)
+          + table.fieldOverride.byName.withProperty('custom.align', 'left')
+          + table.fieldOverride.byName.withProperty('unit', 'ms'),
+          table.fieldOverride.byName.new('HTTP/GRPC rx delay')
+          + table.fieldOverride.byName.withProperty('custom.width', 157)
+          + table.fieldOverride.byName.withProperty('custom.align', 'left')
+          + table.fieldOverride.byName.withProperty('unit', 'ms'),
+          table.fieldOverride.byName.new('HTTP/GRPC tx')
+          + table.fieldOverride.byName.withProperty('custom.align', 'left')
+          + table.fieldOverride.byName.withProperty('unit', 'reqps'),
+          table.fieldOverride.byName.new('HTTP/GRPC rx')
+          + table.fieldOverride.byName.withProperty('custom.align', 'left')
+          + table.fieldOverride.byName.withProperty('unit', 'reqps'),
+          table.fieldOverride.byName.new('HTTP tx success')
+          + table.fieldOverride.byName.withProperty('custom.align', 'left')
+          + table.fieldOverride.byName.withProperty('unit', 'percent'),
+          table.fieldOverride.byName.new('HTTP rx success')
+          + table.fieldOverride.byName.withProperty('custom.align', 'left')
+          + table.fieldOverride.byName.withProperty('unit', 'percent'),
+          table.fieldOverride.byName.new('TCP tx')
+          + table.fieldOverride.byName.withProperty('custom.align', 'left')
+          + table.fieldOverride.byName.withProperty('unit', 'Bps'),
+          table.fieldOverride.byName.new('TCP rx')
+          + table.fieldOverride.byName.withProperty('custom.align', 'left')
+          + table.fieldOverride.byName.withProperty('unit', 'Bps'),
+        ])
+        + table.options.withFooter(
+          table.options.footer.TableFooterOptions.withReducerMixin(['sum'])
+        )
+        + table.queryOptions.withTransformationsMixin([
+        {
+          id: 'merge',
+          options: {}
+        },
+        {
+          id: 'organize',
+          options: {
+            excludeByName: {
+              Time: true,
+            },
+            includeByName: {},
+            indexByName: {
+              Time: 0,
+              'Value #A': 4,
+              'Value #B': 7,
+              'Value #C': 5,
+              'Value #D': 8,
+              'Value #E': 6,
+              'Value #F': 9,
+              'Value #G': 10,
+              'Value #H': 11,
+              cluster: 1,
+              job: 2,
+              service: 3
+            },
+            renameByName: {
+              'Value #A': 'HTTP/GRPC tx',
+              'Value #B': 'HTTP/GRPC rx',
+              'Value #C': 'HTTP/GRPC tx delay',
+              'Value #D': 'HTTP/GRPC rx delay',
+              'Value #E': 'HTTP tx success',
+              'Value #F': 'HTTP rx success',
+              'Value #G': 'TCP tx',
+              'Value #H': 'TCP rx',
```

Comment:
There appears to be in the old dashboard:
              "Value #I": "TCP client closes",
              "Value #J": "TCP client opens",
Did this get removed on purpose?

Reply:
@JonathanWamsley yeah, those were old and not doing anything